packages/nextjs/src/config/loaders/ast.ts

```diff
@@ -320,3 +328,169 @@ export function removeComments(ast: AST): void {
   const nodesWithComments = ast.find(Node).filter(nodePath => !!nodePath.node.comments);
   nodesWithComments.forEach(nodePath => (nodePath.node.comments = null));
 }
+
+/**
+ * Determines from a given AST of a file, whether the file has a default export or not.
+ */
+export function hasDefaultExport(ast: AST): boolean {
+  const hasDefaultDeclaration = ast.find(ExportDefaultDeclaration).size() > 0;
+  if (hasDefaultDeclaration) {
+    return true;
+  }
+
+  const hasDefaultSpecifier = ast.find(ExportDefaultSpecifier).size() > 0;
+  if (hasDefaultSpecifier) {
+    return true;
+  }
+
+  const hasNamedDefaultExport = ast.find(ExportSpecifier).some(specifier => specifier.node.exported.name === 'default');
+  if (hasNamedDefaultExport) {
+    return true;
+  }
+
+  return false;
+}
+
+/**
+ * Extracts all identifiers from a "RestElement" within a named export declaration statement ("export const val = ...").
+ * "RestElements" are things like "...value" inside a destructuring assignment.
```

Comment:
I get that the two `...` actually mean different things here, but that's not totally obvious at first. What about: 

```suggestion
 * Extracts all identifiers from a `RestElement` within a named export declaration statement (`export const constName = constValue`).
 * `RestElements` are things like `...value` inside a destructuring assignment.
```